client/web/compose/src/components/Admin/Module/UniqueValues.vue

```diff
@@ -0,0 +1,179 @@
+<template>
+  <div>
+    <div
+      v-for="(rule, index) in rules"
+      :key="`rule-${index}`"
```

Comment:
if you got 2 indexes dont use generic index name, do a prefix like you did with consIndex